### ISSUE TITLE
Loading world gets stuck

### ISSUE THREAD

**Issue by matyklug18** [Serilum/.issue-tracker#596]
## **Information**
Minecraft version: 1.16.5
Forge version: 36.1.31
Environment: Singleplayer

The Vanilla Experience: 1.1 (from curseforge)
Collective: 2.26 (from curseforge)

## **Description**
When a new world is being created, the game gets stuck, with no output in the logs. Seems like the server thread is stuck at `com.natamus.collective.functions.ItemFunctions#generateEntityDropsFromLootTable` called from `com.natamus.thevanillaexperience.mods.alllootdrops.events.AllLootDropsEntityEvent#onWorldLoad`

I have not yet tested if the same happens if an existing world is being loaded, or if it works with `All Loot Drops` disabled. (currently i have the default config for everything)

Modlist: https://gist.github.com/matyklug18/275debf82c1dfd62822a1b387f43ec36

**Comment by GeorgeOrwell1** [comment]
I've had the same issue using The Vanilla Experience, even after disabling All Loot Drops. Really hopes this gets fixed as it's a great bundle/mod.

**Comment by jburrows82** [comment]
I discovered the same issue with All Loot Drops + MineColonies causes world loading/generation to freeze @ 0%. Both libraries Collective and Structurize can be loaded, but either all loot drops or minecolonies has to be disabled to generate. 

Minecraft 1.16.5 / Forge 36.1.32
All Loot Drops v2.2
Collective v2.26
MineColonies v0.14.252
Structurize v0.13.197

**Comment by GeorgeOrwell1** [comment]
Can confirm it also happens with Realistic Bees.

**Comment by InteJason** [comment]
https://paste.ee/p/EaARw
The report only points to collective, nothing else.

**Comment by FlamenWolf** [comment]
I tried removing minecolonies and it still won't load. I need to remove this mod to get anything to work

**Comment by WolfBV** [comment]
> I discovered the same issue with All Loot Drops + MineColonies causes world loading/generation to freeze @ 0%. Both libraries Collective and Structurize can be loaded, but either all loot drops or minecolonies has to be disabled to generate.
> 
> Minecraft 1.16.5 / Forge 36.1.32
> All Loot Drops v2.2
> Collective v2.26
> MineColonies v0.14.252
> Structurize v0.13.197

Similarly, when All Loot Drops and Advent of Ascension are together, the same issue occurs.

**Comment by matyklug18** [comment]
Seems like creating/loading a world gets stuck even if i disable `All Loot Drops`, `Realistic Bees`, and `Random Village Names`. After disabling all three, it was no longer obvious which part causes it.

**Comment by Disturbeddragon1** [comment]
This happens to me and my husband if we have MineColonies installed even on Minecraft 1.18.1 version. We added our other mods we downloaded back in with the All Loot Drops, Realistic Bees, and Random Village Names mods and it worked so long as we didn't have MineColonies in the mod folder. So seems like All Loot Drops doesn't work well with MineColonies and vice versa. 

This was all tested on the 1.18.1 version of Minecraft with the following:
Forge 1.18.1-39.0.5
All Loot Drops 1.18.1-2.4
MineColonies  1.18.1-1.0.278-RELEASE

I think it may have something to do with MineColonies villagers not having loot drop tables like other mobs as when killed they don't drop anything other than what you gave them to hold in their inventory. I may be wrong in my assumption but that's the only thing I can think of since most mod added mobs tend to have drops that correspond to their type. For instance Alexs Mobs have their own specified drop tables and All Loot Drops doesn't have a problem with it.
